OpenRA.Game/Map/MapCache.cs

```diff
@@ -48,12 +53,22 @@ public MapCache(ModData modData)
 			sheetBuilder = new SheetBuilder(SheetType.BGRA);
 		}
 
+		public void UpdateMaps()
```

Comment:
This can be private, we don't use an access keyword for methods using the default access so just `void UpdateMaps()` is fine.

Reply:
ok